Lab 1/Lab1/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
```

Comment:
This project should be based on .NET Core 3.X
Please, make a rebase for that.

Reply:
Not use .NET Framework as a base for a project.
https://devblogs.microsoft.com/dotnet/net-core-is-the-future-of-net/